pkg/server/structlogging/hot_ranges_log_test.go

```diff
@@ -273,7 +280,7 @@ func setupTest(
 	structlogging.TelemetryHotRangesStatsCPUThreshold.Override(ctx, &st.SV, logCPUThreshold)
 	structlogging.CheckInterval = tickerInterval
 	// wait for the activity from the previous test to drain.
-	time.Sleep(100 * time.Millisecond)
+	time.Sleep(time.Second)
```

Comment:
Do you think we should use testutils.SucceedsSoon? something like:
```
testutils.SucceedsSoon(t, func() error {
  select {
    case structlogging.TestLoopChannel <- struct{}{}:
       return nil
    default:
       return errors.New("Not drained yet")
  }
});

Reply:
It wouldn't fix all the failures. Some of the failures come in the form of this:

```
SucceedsSoon(func() {
    assert.DataIsEmpty(logs)
})
```

If logs trickle in from the last test, the current test will continue to fail. The trick here is trying to figure out how to wait for the settings from the last test to be finally ignored.